specrefs/.ethspecify.yml

```diff
@@ -60,9 +58,18 @@ specrefs:
       - LightClientHeader
       - LightClientOptimisticUpdate
 
-      # Not yet implemented for Gloas
+      # Not implemented: gloas
```

Comment:
We have added all the gloas ones in `containers.yml` apart from `DataColumnSidecar` and `ForkChoiceNode`

Reply:
Cool, can update the specrefs in a bit.